### DISCUSSION TITLE
Tutorial request

### DISCUSSION THREAD

**Discussion by CharlieNash00** [city41/neosdconv#13] [Q&A] [answered]
Hi I'm not familiar with how to use the set of files you uploaded, and I wanted to try to convert some games to .neo format. 
Can you put up a youtube Tutorial showing how it's all done? I tried following your instructions but it didn't work out.

**Comment by city41** [comment, marked as answer]
That's a good idea but I just don't think I have the time for that right now. If you're brand new to node, unfortunately it can be pretty daunting at first. If you are using Windows, it's even worse and I honestly wouldn't be able to help as I don't use Windows. But if you are on Windows, why not use TerraOnion's official tool?

If you are on MacOS or Linux, try this: (and if you are on Windows, it doesn't hurt to try it but you might hit snags)

1. [download Node and install it](https://nodejs.org/en/download/)
2. open Terminal
3. type in `node --version`, if you see something like `v14.17.6` then you are on the right track
4. then `npm install -g neosdconv`
5. now if you enter `neosdconv`, you should see its help screen

From there, the instructions in the README show how to convert a game. So let's say you want to convert KOF94:

1. unzip the kof94.zip file into a directory
2. go into that directory in Terminal
3. `neosdconv -i . -o ./kof94.neo -n 'King of Fighters 94' -g Fighting -y 1994 -m SNK`
4. you should then have `kof94.neo` in that directory, which can be loaded on the NeoSD

**Reply by CharlieNash00** [reply]
Hey this is more helpful, I appreciate your response.

I managed to get the cmd terminal open and when I typed in step > 4. npm install -g neosdconv 

it read out "npm error!  unexpected taken < JSON at position 6

I don't know maybe if I have the directories setup right? I tried forming the files and folders according to the github layout. 

I'll try playing around with it more to see if I have any more success.

**Reply by city41** [reply]
What OS are you using? And what do `node --version` and `npm --version` say?
